linchpin/roles/schedule/tasks/main.yml

```diff
@@ -0,0 +1,73 @@
+---
+
+- set_fact:
+    run_script_path: "{{ run_script_dir_path }}/{{ run_script_name }}"
+    service_file_path: "{{ systemd_dir_path }}/{{ service_name }}.service"
+    timer_file_path: "{{ systemd_dir_path }}/{{ service_name }}.timer"
+  tags:
+    - schedule
+    - clean
+
+- name: Create run script
+  template:
+    src: run_scheduled_tests.sh.j2
+    dest: "{{ run_script_path }}"
+    mode: 0755
+  tags: schedule
+
+- name: Ensure user systemd directory exists
+  file:
+    path: "{{ systemd_dir_path }}"
+    state: directory
+    mode: 0775
+  tags: schedule
+
+- name: Create systemd user service {{ service_file_path }} to run the script
+  template:
+    src: kstests-cloud.service.j2
+    dest: "{{ service_file_path }}"
+    mode: 0664
+  tags: schedule
+
+- name: Create user timer {{ timer_file_path }} to run the service
+  template:
+    src: kstests-cloud.timer.j2
+    dest: "{{ timer_file_path }}"
+    mode: 0664
+  tags: schedule
+
+- name: Restart the timer
+  systemd:
+    name: "{{ service_name }}.timer"
+    user: yes
+    state: restarted
```

Comment:
Don't you also want to enable the timer?

Reply:
Perhaps yes, done